src/Arcus.Observability.Telemetry.Core/TelemetryType.cs

```diff
@@ -1,14 +1,36 @@
-﻿namespace Arcus.Observability.Telemetry.Core
+﻿using Microsoft.Extensions.Logging;
+
+namespace Arcus.Observability.Telemetry.Core
 {
     /// <summary>
-    ///     Different types of telemetry
+    /// Represents the supported types of telemetry available to track during extensions on the <see cref="ILogger"/>.
+    /// Compatible with Azure Application Insights.
```

Comment:
It's compatible with more than Azure Application Insights, no? Maybe rephrase it so that we make it easier to use and add a link to our sink doc?

Reply:
Hmm, yeah, could be used outside AI.

Reply:
Updated this. Thanks!